### PR TITLE
remove InstallWait from create chart code

### PR DESCRIPTION
**Pull request by fgimenez**
Towards https://github.com/giantswarm/chart-operator/issues/73

No need to wait for chart installed, the operator can send the installation request to tiller and continue.

### CODE REVIEW

File: service/controller/v2/resource/chart/create.go [hunk 1]
```diff
@@ -59,8 +59,7 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		//
 		err = r.helmClient.InstallFromTarball(tarballPath, ns,
 			helm.ReleaseName(chartState.ReleaseName),
-			helm.ValueOverrides(values),
-			helm.InstallWait(true))
+			helm.ValueOverrides(values))
```
Comment (MarcelMue) [inline service/controller/v2/resource/chart/create.go]: Maybe an unlikely scenario but what will happen if the install takes longer than the reconciliation period? Will we tell helm to install again even though the last install is still going on?

Comment (fgimenez) [reply service/controller/v2/resource/chart/create.go]: Nope, we return a non-empty create change only if the name of the chart in current state is empty or the name of the chart in the desired state doesn't match the name in the current state, see https://github.com/giantswarm/chart-operator/blob/master/service/controller/v2/resource/chart/create.go#L89.

After sending a request to install tiller will create a release, initially in `PENDING_INSTALL` state. So, in the next reconciliation loop, when we query the current state of the chart here https://github.com/giantswarm/chart-operator/blob/master/service/controller/v2/resource/chart/current.go#L24 the release will be found, either yet in `PENDING_INSTALL` or already in `DEPLLOYED` or `FAILED` state. In any case, the name won't be empty, will match the name of the desired state and the create change will be empty.

Comment (rossf7) [reply service/controller/v2/resource/chart/create.go]: Nice, thanks for the explanation.

**Comment by fgimenez** [comment]
CI green, merging.

**Pull request merged**